### DISCUSSION TITLE
Misspelt variable in ArtifactHub Values file Config

### DISCUSSION THREAD

**Discussion by dmcstravick7** [kedacore/keda#4714] [Q&A / Need Help] [answered]
In the Values file config it references a variable `webhooks.enable` to control the deployment of webhook-admissions. The variable is actually `webhooks.enabled` with the letter 'd' on the end.

Confirmed for a few versions around 2.10.x
https://artifacthub.io/packages/helm/kedacore/keda/2.10.0

**Comment by zroubalik** [comment]
@tomkerkhove FYI

**Comment by tomkerkhove** [comment, marked as answer]
Artifact Hub is just showing our Helm chart README. In 2.11 which is coming soon that should be fixed as they are now automatically generated based on values.yml.

/cc @zroubalik

**Reply by dmcstravick7** [reply]
Thanks for fast turnaround 👍

**Reply by tomkerkhove** [reply]
Happy to help!
